### DISCUSSION TITLE
求助：有道词典笔2系统自动升级和PenMods安装问题

### DISCUSSION THREAD

**Discussion by JohnXX123456** [PenUniverse/PenMods#189] [你问我答]
我最初按照大佬在b站的教程把加强版升级到满分版（2.0.7），然后自己手欠把系统升到了2.1.2。装penmods 时候我又刷了2.0.7系统，但是一打开联网就开始自己下载然后安装2.1.2（必须联网登录后才能进入设置界面关掉更新），我第一次装penmods 的时候从2.1.2刷到了2.0.7系统，也是一打开联网就开始下载然后安装，不过放了几天后又试了一次就稀里糊涂的装上了。今天我装ydpExt没成功，然后就把它在userdisk里的文件删了，可能是因为删错了，重启后就没PenMods，然后系统就从2.0.7升到2.1.2。

我现在主要面临的问题有两个，希望能得到大家的帮助：
    1.刷到2.0.7系统后会自己安装2.1.2系统（重装系统好像没有用，我把系统装回1.6后又装回2.0.7，连网后还是会自己下载安装2.1.2）
     2.装PenMods后但词典笔界面没有变化，再安装时程序显示当前系统已安装PenMods（我个人猜测可能和2.1.2系统已经下载有关）

我对此感到非常困扰，希望能得到大家的帮助和建议。谢谢！

**Comment by EXE-autumnwind** [comment]
我之前就是用2.1.2版本刷的menmods可以正常安装，你试试看行不行

**Reply by JohnXX123456** [reply]
请问您装的什么系统，是PenMods1.2.0吗？

**Reply by tofuture-1** [reply]
可以试试看修改版那个帖子里wxp大佬给的办法

**Reply by JohnXX123456** [reply]
谢谢🙏，我试了六七次后又稀里糊涂的装上了

**Reply by EXE-autumnwind** [reply]
是penmods打错了
